Formula/spatialite-gui.rb

```diff
@@ -24,43 +23,39 @@ class SpatialiteGui < Formula
   depends_on "pkg-config" => :build
   depends_on "freexl"
   depends_on "geos"
-  depends_on "libgaiagraphics"
+  depends_on "libpq"
+  depends_on "librasterlite2"
+  depends_on "librttopo"
   depends_on "libspatialite"
-  depends_on "proj@7"
+  depends_on "libtiff"
+  depends_on "libxlsxwriter"
+  depends_on "lz4"
+  depends_on "minizip"
+  depends_on "openjpeg"
+  depends_on "proj"
   depends_on "sqlite"
+  depends_on "virtualpg"
+  depends_on "webp"
   depends_on "wxwidgets@3.0"
```

Comment:
Did we check if this supports the latest `wxwidgets`?

Reply:
I forgot to mention this, but I tried earlier and it did not work.  In fact the AUR package for `spatialite-gui` is broken because they are trying to use the newest `wxwidgets` and I saw the same error when I tried to upgrade.